### PR TITLE
G Suite migration documentation update

### PR DESCRIPTION
**Pull request by wrall**
Specifically addresses recent Google UI changes, as well as new AADConnect integration limitations.

Addresses:
#2106

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md [hunk 1]
```diff
@@ -95,23 +95,27 @@ Additional migration limitations are described in the following table:
 
 4. On the **Service account permissions (optional)** screen, click **Continue**.
 
-5. Under **Create key (optional)** click **Create Key**.
+5. On the **Grant users access to this service account (optional)** screen, click **Done**.
 
-   ![Create key](../media/gsuite-mig-3-createkey.png)
+6. Once you have been returned to the page listing the service accounts, click on the **Email** for the Service Account you just created to enter the details page, then click the **Edit** button. Alternately, you can click the ellipsis under the **Actions** column and select the **Edit** action.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:100]:
```suggestion
6. Once you have been returned to the page listing the service accounts, click on the **Email** for the Service Account you just created to enter the details page, then click the **Edit** button. Alternatively, you can click the ellipsis under the **Actions** column and select the **Edit** action.
```

File: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md [hunk 2]
```diff
@@ -95,23 +95,27 @@ Additional migration limitations are described in the following table:
...
 
-6. Under **Key type**, make sure **JSON** is selected, and then click **Create**.
+7. On the Service account details page, note the **Unique ID**. This is the ClientId that you will provide later in the instructions for [Grant access to the service account for your Google tenant](#grant-access-to-the-service-account-for-your-google-tenant).
 
-   ![JSON key](../media/gsuite-mig-4-json.png)
+   ![Unique ID](../media/gsuite-mig-6-unique-id.png)
 
-7. Keep track of the JSON keyfile that is automatically downloaded, as you will need its filename during the steps under [Create a migration endpoint in Microsoft 365 or Office 365](#create-a-migration-endpoint-in-microsoft-365-or-office-365). Click **Done**.
+8. If you see an area that says **Show Domain-Wide Delegation**, click to expand that section. Then, enable the checkbox for **Enable G Suite Domain-wide Delegation**, and then click **Save** at the bottom of the page.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:106]:
```suggestion
8. If you see an area that says **Show Domain-Wide Delegation**, click to expand that section. Then, enable the checkbox for **Enable G Suite Domain-wide Delegation** and click **Save** at the bottom of the page.
```

File: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md [hunk 3]
```diff
@@ -95,23 +95,27 @@ Additional migration limitations are described in the following table:
...
+
+   ![Create key](../media/gsuite-mig-3-createkey.png)
 
-9. If you see an area that says **Show Domain-Wide Delegation**, click to expand that section. Then, enable the checkbox for **Enable G Suite Domain-wide Delegation**, and then click **Save**. Google is currently in the process of updating this UI, so you may not see such an option. If you don't see such an option, then this is enabled by default.
+10. Under **Key type**, make sure **JSON** is selected, and then click **Create**.
 
-   ![Domain-Wide Delegation](../media/gsuite-mig-7-enable-domain-delegation.png)
+    ![JSON key](../media/gsuite-mig-4-json.png)
+
+11. Keep track of the JSON keyfile that is automatically downloaded, as you will need its filename during the steps under [Create a migration endpoint in Microsoft 365 or Office 365](#create-a-migration-endpoint-in-microsoft-365-or-office-365). Close the pop-up dialog and click **Save**.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:118]:
```suggestion
11. Keep track of the JSON key file that is automatically downloaded, as you will need its filename during the steps under [Create a migration endpoint in Microsoft 365 or Office 365](#create-a-migration-endpoint-in-microsoft-365-or-office-365). Close the pop-up dialog and click **Save**.
```

File: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md [hunk 4]
```diff
@@ -148,35 +154,33 @@ If your project doesn't already have all of the required APIs enabled, you must
 
 1. Go to the [G Suite Admin page](https://admin.google.com/AdminHome) and sign in as a G Suite admin for your tenant.
 
-2. Click **Domains**, and then **Add/remove domains**, and then click **Add a domain or a domain alias**.
-
-3. Select **Add another domain**. Enter the domain that you will use for routing mails to Microsoft 365 or Office 365. A sub-domain of your primary domain is recommended (such as "o365.fabrikaminc.net" when "fabrikaminc.net" is your primary domain) so that it will be automatically verified. Keep track of the name of the domain you enter because you will need it for the following steps, and later in the instructions as the Target Delivery Domain when you [Create a migration batch in Microsoft 365 or Office 365](#create-a-migration-batch-in-microsoft-365-or-office-365).
+2. Click **Domains**, and then **Manage domains**, and then click **Add a domain**.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:157]:
```suggestion
2. Click **Domains**, then **Manage domains**, and then click **Add a domain**.
```

File: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md [hunk 5]
```diff
@@ -148,35 +154,33 @@ If your project doesn't already have all of the required APIs enabled, you must
...
+5. Log into your DNS provider and update your DNS records so that you have an MX record at the domain you created above in step 3, pointing to Microsoft 365 or Office 365. Ensure that this domain that you created above is an accepted domain in Microsoft 365 or Office 365. Follow the instructions in [Add a domain to Microsoft 365](https://docs.microsoft.com/microsoft-365/admin/setup/add-domain) to add the Microsoft 365 or Office 365 routing domain ("o365.fabrikaminc.net") to your organization and to configure DNS to route mail to Microsoft 365 or Office 365.
 
 ## Create a sub-domain for mail routing to your G Suite domain
 
 1. Go to the [G Suite Admin page](https://admin.google.com/AdminHome) and sign in as a G Suite admin for your tenant.
 
-2. Click **Domains**, and then **Add/remove domains**, and then click **Add a domain or a domain alias**.
+2. Click **Domains**, and then **Manage domains**, and then click **Add a domain alias**.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:173]:
```suggestion
2. Click **Domains**, then **Manage domains**, and then click **Add a domain alias**.
```

File: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md [hunk 6]
```diff
@@ -190,7 +194,7 @@ Once your G Suite environment has been properly configured, you can complete you
...
+You can also deploy Azure Active Directory (Azure AD) Connect to provision your Mail Users, see [Deploy Microsoft 365 Directory Synchronization in Microsoft Azure](https://docs.microsoft.com/office365/enterprise/deploy-office-365-directory-synchronization-dirsync-in-microsoft-azure) for an overview, and [Set up directory synchronization for Microsoft 365](https://docs.microsoft.com/office365/enterprise/set-up-directory-synchronization) for setup instructions. Then, you need to deploy an Exchange server in your on-premises environment for user management, and mail-enable your users using this server. For more information, see [How and when to decommission your on-premises Exchange servers in a hybrid deployment](https://docs.microsoft.com/exchange/decommission-on-premises-exchange) and [Manage mail users](/Exchange/ExchangeServer/recipients/mail-users.md). Once the Mail Users have been created in Microsoft 365, the Azure AD Connect will need to be disabled in order to allow the Migration process to convert these users into Mailboxes - see [Turn off Directory Synchronization](https://docs.microsoft.com/en-us/office365/enterprise/turn-off-directory-synchronization).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:197]:
```suggestion
You can also deploy Azure Active Directory (Azure AD) Connect to provision your Mail Users, see [Deploy Microsoft 365 Directory Synchronization in Microsoft Azure](https://docs.microsoft.com/office365/enterprise/deploy-office-365-directory-synchronization-dirsync-in-microsoft-azure) for an overview, and [Set up directory synchronization for Microsoft 365](https://docs.microsoft.com/office365/enterprise/set-up-directory-synchronization) for setup instructions. Then, you need to deploy an Exchange server in your on-premises environment for user management, and mail-enable your users using this server. For more information, see [How and when to decommission your on-premises Exchange servers in a hybrid deployment](https://docs.microsoft.com/exchange/decommission-on-premises-exchange) and [Manage mail users](/Exchange/ExchangeServer/recipients/mail-users.md). Once the Mail Users have been created in Microsoft 365, the Azure AD Connect will need to be disabled in order to allow the migration process to convert these users into mailboxes - see [Turn off directory synchronization for Microsoft 365](https://docs.microsoft.com/office365/enterprise/turn-off-directory-synchronization).
```

Comment (JohanFreelancer9) [reply Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:197]: Please edit the last sentence as indicated (note capitalization) and remove the locale from the link.

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**